form/form_collections.rst

```diff
@@ -746,6 +746,86 @@ the relationship between the removed ``Tag`` and ``Task`` object.
     you'll need to do extra work to make sure that the relationship is properly
     updated (whether you're adding new tags or removing existing tags) on
     each Tag object itself.
+    
+    You can optionnally reduce the code in your controller and make it reusable to handle other embedded form.
+    First create a helper
+    
+        // App\Helper;
+        use Doctrine\Common\Collections\ArrayCollection;
+        use Doctrine\ORM\EntityManagerInterface;
+        
+        class Helper      
+          private $em;
+
+          public function __construct(EntityManagerInterface $em)
+          {
+            $this->em = $em;
+          }
+          public function backupOriginalEntities($entities)
+          {
+            $original_entities = new ArrayCollection();
+
+                // Create an ArrayCollection of the current objects in the database
+                foreach ($entities as $entity) {
+                    $original_entities->add($entity);
+                }
+
+                return $original_entities;
+          }
+
+          //this function removes only the relationship, removes the entity if $remove set to true
+          public function removeRelation($original_entities, $main_entity,$current_entities, $function_name, $remove=false)
+          {		 
+                foreach ($original_entities as $entity) {
+                    if (false === $current_entities->contains($entity)) {
+                      $main_entity->$function_name($entity);
+
+                      if($remove){
+                        $this->em->remove($entity);
+                      }
+                    }
+                }
+          }
+        }
+        
+   Second edit your TaskController
+   
+        // src/Controller/TaskController.php
+
+        use App\Entity\Task;
+        
+        // ...
+        public function edit($id, Request $request)
+        {
+            $entityManager = $this->getDoctrine()->getManager();
+            $task = $entityManager->getRepository(Task::class)->find($id);
+
+            if (!$task) {
+                throw $this->createNotFoundException('No task found for id '.$id);
+            }
+            
+            $original_entities = $helper->backupOriginalEntities($event->getOrganizationTypes());
+           
+            $editForm = $this->createForm(TaskType::class, $task);
+
+            $editForm->handleRequest($request);
+
+            if ($editForm->isValid()) {
+
+                // remove the relationship between the tag and the Task
+                //depending on your needs               
+                $helper->removeRelation($original_entities,$event,$task->getTags(),'removeTag',true); // or let blank if you want to keep orphan
```

Comment:
You can remove the comment on the end and place that to the comment above;

`// Remove the relationship between the tag and the Task depending on your needs or let blank if you want to keep orphan`

The spelling on this comment is incorrect.